### PR TITLE
Send and receive content uptime messages

### PR DESCRIPTION
**Pull request by alex-deaconu**
Receive data from viewer or schedule
Log to bq
Update as per chrome os player

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc @tejohnso @andrecardoso please review. Thanks!

File: src/main/uptime/uptime.js [hunk 1]
```diff
@@ -39,6 +40,8 @@ function calculate() {
 
     if (!connectedToMS) {investigation.reportConnectivity();}
     uptimeLogger.log(connectedToMS, rendererResult, shouldBePlaying);
+
+    _isActive = connectedToMS && rendererResult && shouldBePlaying;
```
Comment (ezequielc) [inline src/main/uptime/uptime.js]: @alex-deaconu I am wondering if we should remove the `isActive()` check.
We should check uptime regardless if it is `connectedToMS`, to handle offline uptime. 
`shouldBePlaying` is checked in content-uptime directly as we verify if playingItem is not null.
`rendererResult` is the only case that I think we could consider, but it is questionable as well.
Let's talk at standup tomorrow to review. Thanks

**Comment by ezequielc** [comment]
@alex-deaconu Code looks good to me, but I have one discussion point. Thanks!

File: src/main/uptime/uptime.js (hunk 1 above)
Comment (andrecardoso) [reply src/main/uptime/uptime.js]: I agree with @ezequielc , I think these criteria do not apply to content uptime.

Comment (alex-deaconu) [reply src/main/uptime/uptime.js]: @ezequielc @andrecardoso thanks for the feedback. I think it makes sense, removed the `uptime.isActive()` functionality from the code. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc FYI updated this branch as per Chrome OS PR. Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu 👍

**Comment by alex-deaconu** [comment]
@andrecardoso thanks, @tejohnso wondering if you could take a quick look as well before I merge. Thanks!

**Comment by alex-deaconu** [comment]
Thanks!!

**Comment by andrecardoso** [comment]
[(0) Electron -As Player, I provide an uptime logging mechanism for components](https://trello.com/c/AzrraQQU/5374-0-electron-as-player-i-provide-an-uptime-logging-mechanism-for-components)

**Pull request merged**